MotionMark/resources/runner/benchmark-runner.js

```diff
@@ -104,37 +104,40 @@ class BenchmarkRunner {
 
     _runBenchmarkAndRecordResults(state)
     {
-        var promise = new SimplePromise;
-        var suite = state.currentSuite();
-        var test = state.currentTest();
+        const suite = state.currentSuite();
+        const test = state.currentTest();
 
         if (this._client && this._client.willRunTest)
             this._client.willRunTest(suite, test);
 
-        var contentWindow = this._frame.contentWindow;
+        const contentWindow = this._frame.contentWindow;
         var self = this;
```

Comment:
Can we get rid of the one last remaining use of `self` below and remove this as well?